drivers/dma/dma_mcux_edma.c

```diff
@@ -66,6 +66,7 @@ struct dma_mcux_edma_data {
 #define DEV_EDMA_HANDLE(dev, ch)                                               \
 	((edma_handle_t *)(&(DEV_CHANNEL_DATA(dev, ch)->edma_handle)))
 
+
```

Comment:
Maybe we can undo this change as well as there are no changes needed to this file.

Reply:
right, let me update